music.js

```diff
@@ -1,17 +1,28 @@
 // Define the Artist constructor function here, with a 'name' property that can be set at instantiation,
 // and a 'songs' property that is an empty array to start
-function Artist() {
-
+function Artist(name) {
+  this.name = name;
+  this.song = [];
 }
 
 // Define the Song constructor function here, with 'title' and 'artist' properties that can be set at instantiation,
 // and a 'playCount' property that is an empty array to start
-function Song() {
+function Song(title, artist) {
+  this.title = title
+  this.artist = artist
+  this.playCount = 0
 
   // When a new song is instantiated, you'll want to add that song to its artist's 'songs' array.
   // You'll probably want to do that in here somewhere...
-}
+  Artist.prototype.add = function(title) {
```

Comment:
Naming this function `add` is totally acceptable, I'd probably just be a bit more specific next time so I know what exactly the function is adding. For example, addSong as opposed to addAlbum or something.